source/blender/gpu/intern/gpu_shader.c

```diff
@@ -232,6 +232,9 @@ static void gpu_shader_standard_extensions(char defines[MAX_EXT_DEFINE_LENGTH],
 			/* TODO: maybe require this? shaders become so much nicer */
 		}
 	}
+	if (GPU_type_matches(GPU_DEVICE_ANY, GPU_OS_MAC, GPU_DRIVER_ANY) && GPU_legacy_support()) {
```

Comment:
Is this extension needed to be explicitly enabled for shaders ?

Reply:
Not necessary. I will remove